draft-ietf-webtrans-http3.md

```diff
@@ -84,7 +84,8 @@ path value (defined in {{!RFC3986}} Sections 3.2 and 3.3 correspondingly).
 
 When an HTTP/3 connection is established, both the client and server have to
 send a SETTINGS_ENABLE_WEBTRANSPORT setting in order to indicate that they
-both support WebTransport over HTTP/3.
+both support WebTransport over HTTP/3. This process also negotiates the use of
+additional HTTP/3 extensions.
```

Comment:
"both support WebTransport over HTTP/3, including the necessary extensions to the HTTP/3 syntax".